### PR TITLE
Implement subgraph 

### PR DESCRIPTION
**Pull request by wild-mouse**
Implement subgraph for events of new contracts:
- Move subgraph project from inside frontend to root dir
- Add event arguments and marked as indexed to filter events without pure web3 library when necessary
- Remove noneed event arguments to reduce gas fees and subgraph can fetch these data from contract directly
- Implement subgraph mappings

- [x] Deploy subgraph on local and verify operation

### CODE REVIEW

File: subgraph/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,73 @@
+type Token @entity {
+  id: ID!
+  creator: Creator! # address
+  name: String! # string
+  symbol: String!
+  totalSupply: BigInt!
+  decimals: Int
+  creatorTokenRatio: Int
+  isTotalSupplyFixed: Boolean
```
Comment (toshiakitakase) [inline subgraph/schema.graphql:9]: All tokens created in the issuance phase are mintable, so I think `isTotalSupplyFixed` flag is unnecessary.

File: subgraph/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  lockupPeriod: Int
+  enableStakeToToken: Boolean
```
Comment (toshiakitakase) [inline subgraph/schema.graphql:11]: I thought `enableStakeToToken` was also unnecessary because we don't currently plan on staking.

**Comment by wild-mouse** [comment]
Implemented getting checking request status.

- subgraph tracks Chainlink request events and creates or updates an entity.
- When a request is fulfilled, subgraph calls the campaign contract and gets claimable status, and set to the entity.

https://github.com/TART-tokyo/iroiro/blob/34b763da6a88df5b4a02970f56de07a8071d6b63/subgraph/schema.graphql#L44-L51

**Comment by wild-mouse** [comment]
- Verified operation on local and fixed some issues 
- Downgraded node version due to truffle error

**Review by toshiakitakase — APPROVED** [review]
LGTM

**Pull request merged**